rolling-hash.md

```diff
@@ -1,11 +1,11 @@
 # Rolling Hash Algorithm
-_Spec v4 (2021-03-09)_
+_Spec v5 (2022-04-04)_
 
-Make a rolling hash based file diffing algorithm. When comparing original and an updated version of an input, it should return a description ("delta") which can be used to upgrade an original version of the file into the new file. The description contains the chunks which:
+Make a rolling hash based file diffing algorithm. When comparing original and an updated version of an input, it should return a description ("delta") which can be used to upgrade an original version of the file into the new file. The description provides information of the chunks which:
```

Comment:
Earlier wording here, with the first point, was contradicting with the third point of requirements, where *only* the changes should be returned in delta.